configure.ac

```diff
@@ -62,7 +62,7 @@ AX_PTHREAD([], [
 	       echo "Error! We require pthreads to be available"
 	       exit -1
 	       ])
-LIBS="$PTHREAD_LIBS $LIBS"
+OFP_LIBS="$PTHREAD_LIBS $OFP_LIBS"
```

Comment:
What ODP-DPDK/DPDK version and configuration options are you using? I'm just wondering since I have not seen this issue before.

Reply:
dpdk: 18.11.0
odp-dpdk:
commit d25d47b3c5b1dcde440960c259d5066c2dec0a2d
./configure --with-dpdk-path=/home/bogdan/Work/build/dpdk/install --prefix=/home/bogdan/Work/bogdanPricope/odp-dpdk_install

Problem was observed when starting native applications with netwarp libraries (so, not during build) due to the fact that netwrap libraries were also linked with DPDK libs.

Reply:
There have been some pkg-config changes in the latest odp-dpdk code (after d25d47b3c) and this PR seems to break the build when odp-dpdk is configured with `--disable-abi-compat` flag.

```
make[2]: Entering directory '/home/nnn/dev/ofp.git/test'
  CXX      cpp_test.o
  CC       redefinitions_posix_test.o
  CCLD     redefinitions_posix_test
  CXXLD    cpp_test
../lib/.libs/libofp.a(ofp_init.o): In function `rte_pktmbuf_prefree_seg':
/home/nnn/dev/dpdk.git/install/include/dpdk/rte_mbuf.h:1691: undefined reference to `per_lcore__lcore_id'
../lib/.libs/libofp.a(ofp_init.o): In function `rte_mempool_ops_enqueue_bulk':
/home/nnn/dev/dpdk.git/install/include/dpdk/rte_mempool.h:704: undefined reference to `rte_mempool_ops_table'

```

Reply:
Targeting 'HEAD' of odp-dpdk will make ofp unstable. However, the problem is also in my version (with --disable-abi-compat) so I'll (try to) make a fix.

Reply:
This should be fixed now. Do you still have problems with build?

Reply:
The issue still persists.
```
./configure --with-odp=/home/nnn/odp-dpdk_install --with-cunit-path=/home/nnn/CUnitHome --enable-sp=no
make
...
make[2]: Entering directory '/home/nnn/dev/ofp.git/test/cunit'
  CC       ofp_test_arp.o
  CCLD     ofp_test_arp
../../lib/.libs/libofp.a(ofp_init.o): In function `rte_pktmbuf_prefree_seg':
/home/nnn/dev/dpdk.git/install/include/dpdk/rte_mbuf.h:1183: undefined reference to `per_lcore__lcore_id'
../../lib/.libs/libofp.a(ofp_init.o): In function `rte_mempool_ops_enqueue_bulk':
/home/nnn/dev/dpdk.git/install/include/dpdk/rte_mempool.h:786: undefined reference to `rte_mempool_ops_table'

```

Reply:
Ok.. changed.